emulators/qemu/Portfile

```diff
@@ -25,11 +25,12 @@ homepage                https://www.qemu.org
 master_sites            https://download.qemu.org/
 use_xz                  yes
 
-checksums               rmd160  01c82bed8b0219935c00981e16b573004b3802d0 \
-                        sha256  a0634e536bded57cf38ec8a751adb124b89c776fe0846f21ab6c6728f1cbbbe6 \
-                        size    121833004
+checksums               rmd160  50661dd45f91d900241e0e6b919140a6b8a2b2cf \
+                        sha256  5b49ce2687744dad494ae90a898c52204a3406e84d072482a1e1be854eeb2157 \
+                        size    122408576
 
 depends_build           port:texinfo \
+                        port:libslirp \
```

Comment:
This is good, but we should also add an explicit `--enable-slirp` to the `configure.args` to ensure we always link with libslirp and do not rely on the auto-detection of the configure script.

This is small enough that we do not need it as a variant (user networking is so common that I doubt anyone really needs to deselect it for minimal gains).